src/cross_section.F90

```diff
@@ -478,18 +461,17 @@ subroutine calculate_urr_xs(i_nuclide, E)
     ! Determine treatment of inelastic scattering
     inelastic = ZERO
     if (urr % inelastic_flag > 0) then
-      ! Get pointer to inelastic scattering reaction
-      rxn => nuc % reactions(nuc % urr_inelastic)
-
       ! Get index on energy grid and interpolation factor
       i_energy = micro_xs(i_nuclide) % index_grid
       f = micro_xs(i_nuclide) % interp_factor
 
       ! Determine inelastic scattering cross section
-      if (i_energy >= rxn % threshold) then
-        inelastic = (ONE - f) * rxn % sigma(i_energy - rxn%threshold + 1) + &
-             f * rxn % sigma(i_energy - rxn%threshold + 2)
-      end if
+      associate (rxn => nuc % reactions(nuc % urr_inelastic))
+        if (i_energy >= rxn % threshold) then
```

Comment:
This will be my last comment about inconsistent whitespace around `%`, I simply think a standard style should be defined and adhered to (or aspired to at least).